kolibri/plugins/management/assets/src/views/class-edit-page/user-remove-modal.vue

```diff
@@ -72,13 +70,35 @@
         required: true,
       },
     },
+    computed: {
+      formattedDeleteConfirmation() {
+        return this.$tr(
+          'deleteConfirmation',
+          {
+            username: this.bold(this.username),
```

Comment:
since `bold` doesn't use `this`, it doesn't have to be a `methods` member. It can be defined as a helper at the module level (between `const actions =` and `module.exports =`).